README.md

```diff
@@ -2,16 +2,43 @@
 [![code coverage](https://img.shields.io/codecov/c/gh/Molara-Lab/Molara)](https://codecov.io/gh/Molara-Lab/Molara)
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/Molara-Lab/Molara/main.svg)](https://results.pre-commit.ci/latest/github/Molara-Lab/Molara/main)
 # Molara
+Molara is an open-source program for the 3-dimensional visualization of molecules and crystal structures. These are some of its main features:
+
+1. Import of .xyz, .coord, and POSCAR files
+2. Export of rendered structures as raster graphics
+3. Tools for creating custom molecular and crystal structures
+4. Animation of trajectories
+
+New features will follow soon!
+
+## Installation
+You first need to clone the repository:
 ```
 git clone <this repo>
 cd Molara
 ```
-You first need to create a virtual environment and activate it in order to install molara as follows:
+
+It is advisable to install Molara in a virtual Python environment.
+
+<b>Virtual environment on Linux / Mac:</b>
 ```
+python -m pip install virtualenv
```

Comment:
Does one actually need `virtualenv` to create a virtual environment? I thought `venv` was part of standard python since 3.3.

Reply:
I wasn't aware of that, thanks for pointing it out!